Tests/MacTests.swift

```diff
@@ -22,74 +22,74 @@ import Foundation
 import XCTest
 import Valet
 
-
-class ValetMacTests: XCTestCase
-{
-    // This test verifies that we are neutralizing the zero-day Mac OS X Access Control List vulnerability.
-    // Whitepaper: https://drive.google.com/file/d/0BxxXk1d3yyuZOFlsdkNMSGswSGs/view
-    // Square Corner blog post: https://corner.squareup.com/2015/06/valet-beats-the-ox-x-keychain-access-control-list-zero-day-vulnerability.html
-    @available (OSX 10.10, *)
-    func test_setStringForKey_neutralizesMacOSAccessControlListVuln()
+#if os(macOS)
+    class ValetMacTests: XCTestCase
```

Comment:
Nit: What do you think about leaving this un-indented. Feels kinda weird to indent an entire class just because it's conditionally compiled.

Reply:
Hard agree that it's weird. Indentation was the default on Xcode 9 with ctrl-i. On Xcode 10 the indentation goes away, so I'll go with that :)